sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -176,6 +176,17 @@ extern "C" {
 #define CONFIG_GNRC_IPV6_NIB_QUEUE_PKT                1
 #endif
 
+#if CONFIG_GNRC_IPV6_NIB_QUEUE_PKT
+/**
+ * @brief    queue capacity for the packets waiting for address resolution,
+ *           per neighbor. SHOULD always be smaller than @ref CONFIG_GNRC_IPV6_NIB_NUMOF
+ */
+#ifndef CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP
+#define CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP  (CONFIG_GNRC_IPV6_NIB_NUMOF > 16 ? 16 : 1)
```

Comment:
that drop is a bit strange. Up to 16 entries, each entry can queue only a single packet, but once there are 17 entries, we can queue 16 per entry?

How about 
```suggestion
#define CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP  DIV_ROUND_UP(CONFIG_GNRC_IPV6_NIB_NUMOF, 4)
```

Reply:
I guess it's fine, I don't know how probable is resolution of multiple neighbors at the same time.